### PR TITLE
[CUBRIDMAN-24] add query-output and loaddb-output option to csql utility

### PR DESCRIPTION
**Pull request by mhoh3963**
- change usage of csql
- change an example of -t option
- add a description and an example of -q option and -d option
- change an example of -N option
- add a description of --delimiter and --enclosure

### CODE REVIEW

File: en/csql.rst [hunk 1]
```diff
@@ -221,38 +225,64 @@ To display the option list in the prompt, execute the **csql** utilities without
...
+
+        code    name    gender  nation_code event
+        12762   O'Brien Dan M   USA Athletics
+        12763   O'Brien Leah    W   USA Softball
+        12764   O'Brien Shaun William   M   AUS Cycling
+        12765   O'Brien-Amico Leah  W   USA Softball
  
+.. option:: -q, --query-output
+
+    This option displays the result for easy use in insert queries, only show column names and values and works with **-c** or **-i** option. Each column name and value is separated by a comma or a single character of the **--delimiter** option and all results except for numeric types is enclosed by a single quote or a single character of the **--enclosure** option. If the enclosure is single quote, the single quote in the results is replaced with two ones. It is ignored when it is given with **-l** option.
```
Comment (charischau) [inline en/csql.rst]: 1. only show column names and values and works with **-c** or **-i** option --> only show column names and values, and works with **-c** or **-i** option (comma added)

2. Each column name and value is separated by a comma or a single character of the **--delimiter** option and all results except for numeric types is enclosed by a single quote or a single character of the **--enclosure** option.  

- Each column name and value is...  --> Each column name and value are... (is changes to are)

- The sentence is too long so maybe adding ';'  before  'and all results except for numeric types...'

- all results except for numeric types is enclosed by ... --> all results except for numeric types are enclosed by ( is --> are)

3.  If the enclosure is single quote --> If the enclosure is a single quote ('a' added)

File: en/csql.rst [hunk 2]
```diff
@@ -221,38 +225,64 @@ To display the option list in the prompt, execute the **csql** utilities without
...
+
+        code,name,gender,nation_code,event
+        12762,"O'Brien Dan","M","USA","Athletics"
+        12763,"O'Brien Leah","W","USA","Softball"
+        12764,"O'Brien Shaun William","M","AUS","Cycling"
+        12765,"O'Brien-Amico Leah","W","USA","Softball"
+
+.. option:: -d, --loaddb-output
+
+    This option displays the result for easy use in loaddb utility, only show column names and values and works with **-c** or **-i** option. Each column name and value is separated by a space and all results except for numeric types is enclosed by a single quote. The single quote in the results is replaced with two ones and the result of the enum type is outputed by an index instead of a value. This opton is ignored when it is given with **-l** option.
```
Comment (charischau) [inline en/csql.rst]: 1. only show column names and values and works with **-c** or **-i** option. --> only show column names and values, and works with **-c** or **-i** option. (comma added)

2. Each column name and value is separated by a space and all results except for numeric types is enclosed by a single quote.

- Each column name and value is... --> Each column name and value are... (is --> are)

- The sentence is too long so maybe adding ';' before 'and all results except for numeric types...'

- all results except for numeric types is enclosed by ... --> all results except for numeric types are enclosed by ( is --> are)

3. the result of the enum type is outputed --> the result of the enum type is outputted (past perfect tense of output is outputted)

File: en/csql.rst (hunk 1 above)
Comment (mhoh3963) [reply en/csql.rst]: thank you for reviewing, i will apply your comment.

File: en/csql.rst (hunk 2 above)
Comment (mhoh3963) [reply en/csql.rst]: thank you for reviewing, i will apply your comment.

**Review by charischau — DISMISSED** [review]

File: ko/csql.rst [hunk 3]
```diff
@@ -221,38 +225,64 @@ CSQL 시작 옵션
...
+
+        code,name,gender,nation_code,event
+        12762,"O'Brien Dan","M","USA","Athletics"
+        12763,"O'Brien Leah","W","USA","Softball"
+        12764,"O'Brien Shaun William","M","AUS","Cycling"
+        12765,"O'Brien-Amico Leah","W","USA","Softball"
+
+.. option:: -d, --loaddb-output
+
+    결과를 loaddb 유틸리티에서 사용할 수 있게 출력하는 옵션으로 컬럼명과 값만 표시되며 **-c** 또는 **-i** 옵션과 같이 사용해야 한다. 각 컬럼명과 값은 공백으로 구분되며, 숫자형 타입를 제외한 모든 결과들은 작은 따옴표로 둘러싸여 출력된다.  결과안의 작은 따옴표는 두개 작은 따옴표로 대체되며, 열거형(ENUM)타입의 결과는 값대신 색인값을 출력한다. 이 옵션은 **-l** 옵션과 함께 지정된 경우에는 무시된다.
```
Comment (kisoo-han) [inline ko/csql.rst]: 띄어쓰기:
값대신 -> 값 대신

**Review by kisoo-han — DISMISSED** [review]

File: ko/csql.rst (hunk 3 above)
Comment (mhoh3963) [reply ko/csql.rst]: thank you, i applied your comment.

**Review by kisoo-han — APPROVED** [review]

**Pull request merged**